lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -35,7 +37,13 @@ def call(env)
           # start a new request span and attach it to the current Rack environment;
           # we must ensure that the span `resource` is set later
           request_span = tracer.trace('rack.request', trace_options)
-          env[:datadog_rack_request_span] = request_span
+          env[RACK_REQUEST_SPAN] = request_span
+
+          # For backwards compatibility; remove later.
+          env[:datadog_rack_request_span] = env[RACK_REQUEST_SPAN] 
```

Comment:
good to me, but let's add a `TODO:` just for the highlight. Also we should probably think (before shipping / merging this part) to write in the comment: `remove later` -> `this attribute is deprecated and will be removed in 0.13` (it's just an example).

Reply:
Yeah, good idea. I will add in the "will be removed in 0.13" for now, and if we change the plan, we can adjust it later.